.env.example

```diff
@@ -8,4 +8,5 @@ CLIENT_REPORT_MAX_PROVIDER_DEAL_PERCENTAGE=25
 CLIENT_REPORT_MAX_DUPLICATION_PERCENTAGE=20
 CLIENT_REPORT_MAX_PERCENTAGE_FOR_LOW_REPLICA=25
 CLIENT_REPORT_MAX_LOW_REPLICA_THRESHOLD=3
-PROMETHEUS_AUTH_TOKEN="placeholder"
+DATACAPSTATS_API_KEY="some-key"
```

Comment:
@lukasz-wal let's make infra ready for this

Reply:
it was only missing in .env.example file and was already being used